src/ImageSharp/ImageSharp.csproj

```diff
@@ -5,6 +5,7 @@
     <VersionPrefix Condition="$(packageversion) != ''">$(packageversion)</VersionPrefix>
     <VersionPrefix Condition="$(packageversion) == ''">0.0.1</VersionPrefix>
     <Authors>Six Labors and contributors</Authors>
+    <TargetFrameworks>netstandard1.1;netstandard1.3;netstandard2.0</TargetFrameworks>
```

Comment:
Duplicate `<TargetFrameworks/>` declarations. What were the requirements for 2.0?

Reply:
You can keep  .NET Standard 1.0 and 1.3 backends.

I tried .NET Standard 2.0 for libraries and it worked OK.

I also used .NET Standard 6.1 for the "ImageSharp Picture Viewer".

All the code (yours and mine) is compatible with all frameworks: Standard for the libraries and Core for the apps except "ImageSharp Picture Viewer", that requires Framework/Mono for the GUI.